proto/viam/app/packages/v1/packages.proto

```diff
@@ -92,20 +92,6 @@ message Package {
   string id = 5;
 }
 
-// InternalPackage is stored in the packages database and represents our interval view of the uploaded package
```

Comment:
Replaced by [dbPackage](https://github.com/viamrobotics/app/blob/4e313dba532bb2bb2a817027f1c94bc08ecaa362/packages/db.go#L76-L90).

Reply:
Now that we've removed `InternalPackage`, seems like we need to remove the import "tagger/v1/tagger.proto"